### PR TITLE
Create accts.transactions.mdx

### PR DESCRIPTION
**Pull request by kbinreallife**

### CODE REVIEW

File: docs/scripting/trust scripts/accts.transactions.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+title: accts.transactions
+---
+
+## Description
+
+accts.transactions returns a log of the user's transactions.
+
+## Syntax
+
+On the Command Line Interface:
```
Comment (ghambhackmud) [inline docs/scripting/trust scripts/accts.transactions.mdx:11]:
```suggestion
### CLI
```

Brings inline with template

File: docs/scripting/trust scripts/accts.transactions.mdx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+```
+accts.transactions
+```
+
+As a subscript:
```
Comment (ghambhackmud) [inline docs/scripting/trust scripts/accts.transactions.mdx:17]:
```suggestion
### Script
```

Brings inline with template

File: docs/scripting/trust scripts/accts.transactions.mdx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+### Parameters
+
+#### count
+
+The 'count' argument allows a user to specify the number of transactions to be returned.
+
+#### start
+
+The 'start' argument can be used in combination with 'count', or on it's own, to offset the beginning of the list to the desired position.
```
Comment (ghambhackmud) [inline docs/scripting/trust scripts/accts.transactions.mdx:31]:
```suggestion
The 'start' argument offsets the beginning of the list of transactions to the desired position. Can be used in combination with 'count'.
```

Comment (ghambhackmud) [reply docs/scripting/trust scripts/accts.transactions.mdx:31]: Maybe less confusing wording?

File: docs/scripting/trust scripts/accts.transactions.mdx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+#### to
+
+A user may filter transactions that are sent to a specific username.
```
Comment (ghambhackmud) [inline docs/scripting/trust scripts/accts.transactions.mdx:35]:
```suggestion
The 'to' argument filters transactions that are sent to a specific username.
```

File: docs/scripting/trust scripts/accts.transactions.mdx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+#### from
+
+A user may filter transactions that are sent from a specific username.
```
Comment (ghambhackmud) [inline docs/scripting/trust scripts/accts.transactions.mdx:39]:
```suggestion
The 'from' argument filters transactions that are sent from a specific username.
```

File: docs/scripting/trust scripts/accts.transactions.mdx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+#### script
+
+A user may filter transactions by the specific script which sent them.
```
Comment (ghambhackmud) [inline docs/scripting/trust scripts/accts.transactions.mdx:43]:
```suggestion
The 'script' argument filters transactions by the specific script which sent them.
```

**Review by ghambhackmud — CHANGES_REQUESTED** [review]
Suggested changes that bring document inline with template.

Also suggested some wording changes, some keeping the previous style of the article.

**Review by ghambhackmud — APPROVED** [review]
Looks good

**Review by ghambhackmud — APPROVED** [review]
Looks good

**Review by Fayti1703 — APPROVED** [review]
Not sure if we'd want to mention how the special-sauce output produced in CLI-mode works (which is what you get with `is_script: false`)

**Pull request merged**